src/pages/answer/resultsTable/ResultExplorer.jsx

```diff
@@ -350,7 +350,7 @@ export default function ResultExplorer({ answerStore }) {
       elevation={3}
     >
       <h5 className="cardLabel">Answer Explorer</h5>
-      {answerStore.showNodePruneSlider && (
+      { Boolean(answerStore.showNodePruneSlider)  && (
```

Comment:
Actually, I just saw, this in the CI/CD lint checker. Looks like there are two spaces here:

![image](https://github.com/RobokopU24/qgraph/assets/16181779/a5c80c04-2d0b-4b52-848f-1f06fab7c48f)

You can run `npm run lint` to check the whole project before pushing, otherwise it won't let us merge 🙄. Also, install the "ESLint" plugin in VSCode so it will highlight these issues for you. To fix this one, just submit another commit and it will rerun the CI/CD.

Reply:
Ok great! Yeah I was confused about this. Thanks!